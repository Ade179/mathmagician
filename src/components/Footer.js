import React, { PureComponent } from 'react';
import './CSSmodules/footer.css';
import { Link } from 'react-router-dom';

class Footer extends PureComponent {
  render() {
    return (
      <>
        <section className="thesect">
          <div className="logo">
            <h1 className="magicians">Math Magicians</h1>
          </div>
          <div>
            <h2 className="maker">Created by Daniel</h2>
          </div>
          <nav className="nav">
            <Link className="links" to="/">Home</Link>
            |
            <Link className="links" to="/calculator">Calculator</Link>
            |
            <Link className="links" to="/quotes">Quotes</Link>
          </nav>
        </section>
      </>
    );
  }
}

export default Footer;
