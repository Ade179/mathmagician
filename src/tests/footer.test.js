import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

test('should match the snapshot', () => {
  const home = renderer.create(<Footer />);
  const tree = home.toJSON();
  expect(tree).toMatchSnapshot();
});
