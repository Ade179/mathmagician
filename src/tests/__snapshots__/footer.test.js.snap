// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<section
  className="thesect"
>
  <div
    className="logo"
  >
    <h1
      className="magicians"
    >
      Math Magicians
    </h1>
  </div>
  <div>
    <h2
      className="maker"
    >
      Created by Daniel
    </h2>
  </div>
  <article
    className="contact"
  >
    <div
      className="contacts"
    >
      Conatct:+2348114374848
    </div>
  </article>
</section>
`;
